apps/fluent-tester/src/TestComponents/Text/V2Usage.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Platform, Text, View } from 'react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import {
+  Body1,
+  Body1Strong,
+  Body2,
+  Body2Strong,
+  Caption1,
+  Caption1Strong,
+  Caption2,
+  Display,
+  LargeTitle,
+  Title1,
+  Title2,
+  Title3,
+} from '@fluentui-react-native/text';
+
+export const V2Usage: React.FunctionComponent = () => {
+  if (Platform.OS === 'ios') {
+    return (
+      <View>
+        <Stack style={stackStyle} gap={5}>
+          <Caption2>Caption2</Caption2>
+          <Caption1>Caption1</Caption1>
+          <Caption1Strong>Caption1Strong</Caption1Strong>
+          <Body2>Body2</Body2>
+          <Body2Strong>Body2Strong</Body2Strong>
+          <Body1>Body1</Body1>
+          <Body1Strong>Body1Strong</Body1Strong>
+          <Title3>Title3</Title3>
+          <Title2>Title2</Title2>
+          <Title1>Title1</Title1>
+          <LargeTitle>LargeTitle</LargeTitle>
+          <Display>Display</Display>
+        </Stack>
+      </View>
+    );
+  } else {
+    return <Text>Only available on iOS for now.</Text>;
+  }
```

Comment:
Would prefer we try and support the V1 API, rather than make a V2 that's only on iOS.

Reply:
913342d introduces a better approach for this.